docs/source/example_npe_model.md

```diff
@@ -0,0 +1,127 @@
+steps are the essentially same as [the toy example](example_toy_model.md) but with higher level settings. It is
+recommended to run this on a cluster or GPU machine. 
+
+We can repeat the same first few steps from the previous tutorial with a couple differences. 
+
+
+Step 1 Generating a Waveform Dataset
+------------------------------------ 
+
+As before we run `dingo_generate_dataset`:
+
+```
+cd dingo
+mkdir $(pwd)/npe_model_train_dir
+export TRAIN_DIR=$(pwd)/npe_model_train_dir
+dingo_generate_dataset --settings examples/waveform_dataset_settings.yaml --out_file $TRAIN_DIR/waveform_dataset.hdf5
+```
+
+The `examples/npe_model/waveform_dataset_settings.yaml` settings file now includes a new attribute `compression`.
+This creates a truncated singular value decomposition (SVD) of the waveform polarizations which 
+is stored on disk as a compressed representation of the dataset. The `size` attribute 
+refers to the number of basis vectors included in the expansion of the waveform. This can later be
+changed during training. When the compression phase is finished, the log will
+display the mismatch between the decompressed waveform and generated waveform. You can
+also access these mismatch settings by running `dingo_ls` on a generated `waveform_dataset.hdf5`
+file. It will show multiple mismatches corresponding to the number of basis vectors used
+to decompress the waveform. It is up to the user as to what type of mismatch is acceptable,
+typically a maximum mismatch of $10^{-3}-10^{-4}$ is recommended. 
+
+We could also generate the waveform dataset using a
+[condor DAG](https://htcondor.readthedocs.io/en/latest/users-manual/dagman-workflows.html)
+on a cluster. To do this run
+
+```
+dingo_generate_dataset_dag --settings_file $(pwd)/examples/npe_model/waveform_dataset_settings.yaml --out_file $TRAIN_DIR/IMRPhenomXPHM.hdf5 --env_path $DINGO_VENV_PATH --num_jobs 4 --request_cpus 64 --request_memory 128000 --request_memory_high 256000
+```
+
+and then submit the generated DAG
+
+```
+condor_submit_dag condor/submit/dingo_generate_dataset_dagman_DATE.submit
+```
+
+where `DATE` is specified in the filename of the `.submit` file that was generated.
+
+
+Step 2 Generating an ASD dataset
+--------------------------------
+
+To generate an ASD dataset we can run the same command as in the previous tutorial.
+
+```
+dingo_generate_asd_dataset --settings_file examples/npe_model/asd_dataset_settings_fiducial.yaml --data_dir $TRAIN_DIR/asd_dataset_folder -out_name $TRAIN_DIR/asds_O1_fiducial.hdf5
+```
+
+However, this time, during training we will need two sets of ASDs. The first one will be
+fixed during the initial training -- this is the fiducial dataset generated above.
+This dataset will contain only  a single ASD. The second ASDDataset will contain many
+ASDs and is used during the fine tuning stage. To generate this second dataset run
```

Comment:
This will seem somewhat mysterious to a reader. Why are we using two sets of ASDs?

Reply:
https://github.com/dingo-gw/dingo/pull/173/commits/fad0ea2cd785ec8978b6aba2f5703c83e635eb8c